docs/13_upload.md

```diff
@@ -90,53 +96,89 @@ two different servers differ slightly.
             dandi validate .
             dandi upload
 
-    Note that the `organize` steps should not be used if you are preparing a BIDS dataset with the NWB files.
-    Uploading to the development server is controlled via `-i` option, e.g.
-    `dandi upload -i dandi-staging`.
-    Note that validation is also done during `upload`, but ensuring compliance using `validate` prior upload helps avoid interruptions of the lengthier upload process due to validation failures.
-    6. Add metadata by visiting your Dandiset landing page:
+        - Note that the `organize` steps should not be used if you are preparing a BIDS dataset with the NWB files.
+        - Uploading to the development server is controlled via `-i` option, e.g.
+        `dandi upload -i dandi-staging`.
+        - Note that validation is also done during `upload`, but ensuring compliance using `validate` prior upload helps avoid interruptions of the lengthier upload process due to validation failures.
+        - If you have an issue using the `dandi` CLI, see the [Dandi Debugging section](./15_debugging.md).
+    
+    1. Add metadata by visiting your Dandiset landing page:
        `https://dandiarchive.org/dandiset/<dataset_id>/draft` and clicking on the `METADATA` link.
 
-If you have an issue using the Python CLI, see the [Dandi Debugging section](./15_debugging.md).
+1. BIDS format
+    1. Convert your data to BIDS format in a local folder. Let's call this `<source_folder>`.
+    We suggest beginning the conversion process using only a small amount of data so that common issues may be spotted earlier in the process.
+    This step can be complex depending on your data.
+    [BIDS converters](https://bids.neuroimaging.io/benefits.html#converters)
+    automates conversion to BIDS from a variety of popular formats and the [BIDS Specification](https://bids-specification.readthedocs.io/) provides more information on the BIDS standard.
+    Feel free to [reach out to us for help](https://github.com/dandi/helpdesk/discussions).
+
+    1. Once your files are confirmed to adhere to the best practices, perform an official validation of the BIDS files by running:
```

Comment:
For completeness, since our BIDS validation is too rudimentary, we are doomed to advise people to first validate using official BIDS validator:  https://github.com/bids-standard/bids-validator

Reply:
Good point.  I have updated in this [commit](https://github.com/dandi/handbook/pull/108/commits/afed1b6a4c190eef7f0a72d0621355ece6b1d75a).